native-authentication/src/main/java/com/okta/nativeauthentication/form/FormFactory.kt

```diff
@@ -35,6 +36,11 @@ internal class FormFactory(
             }
         }
 
+        if (previousEmission.get() == formBuilder) {
+            return
+        }
```

Comment:
Why is this needed?

Reply:
It's not strictly necessary. I did it because I wanted to skip double emissions of loading.

Due to abstractions, we emit loading any time we make a network call. But sometimes we need those emissions in multiple places for things like retries. So rather than complicate how we emit those events, I just decided to filter them out here instead.

Reply:
That being said, it would have still worked just fine with the double emission, it just made the tests confusing. But Compose UI would have basically filtered out the double emission at the UI level too.

Reply:
Gotcha, thanks for clarifying!